rest/data/interface.go

```diff
@@ -399,3 +399,17 @@ type Connector interface {
 	// CompareTasks returns the order that the given tasks would be scheduled, along with the scheduling logic.
 	CompareTasks([]string, bool) ([]string, map[string]map[string]string, error)
 }
+
+type MockConnector interface {
```

Comment:
Why do we still have this interface? It seems like MockDBConnector and MockDBConnectorImpl is what we want. If it seems like some amount of work to get MockDBConnector to just implement Connector (and remove this mock interface) we could break that into a different ticket since tests are passing.

Reply:
You're right we don't need this interface, removing it entirely doesnt cause any errors. MockDBConnector already implements Connector.